Resources/Textures/_Harmony/Clothing/Uniforms/Jumpsuit/hop.rsi/meta.json

```
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Originally taken from tgstation atcommithttps://github.com/tgstation/tgstation/commit/c838ba21dae97db345e0113f99596decd1d66039 icon, inhand, and equipped resprites by CrimsonJupiter for Harmony.",
```

Multiple spaces missing in copyright
```suggestion
  "copyright": "Originally taken from tgstation at commit https://github.com/tgstation/tgstation/commit/c838ba21dae97db345e0113f99596decd1d66039 icon, inhand, and equipped resprites by CrimsonJupiter for Harmony.",
```